### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by maov19**
The following features were added:

- Created a collection that keeps a list of books and display them in the top of the page.
- Created a function to add a new book to the collection, with title and author.
- Created a function to remove a book from the collection.
- Data is preserved in the browser's memory by using local storage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,162 @@
...
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
```
Comment (muneebulrehman) [inline README.md]: Kindly remove this commented out part from your README file to keep it clean and professional.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,162 @@
...
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```
Comment (muneebulrehman) [inline README.md]: Kindly remove this part of the README template. This is irrelevant to the project and is only for reference when you use the README template provided by the microverse.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- All technical requirements met for this milestone.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (maov19) [reply README.md]: Fixed! 🔧

File: README.md (hunk 1 above)
Comment (maov19) [reply README.md]: Fixed! 🔧

**Comment by maov19** [comment]
Hi! README lines 1-40 were removed as requested.
Please let us know if there is something else to work on.

**Review by Munsa1 — APPROVED** [review]
Hi @App-Developer8866 @maov19 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made required changes on ReadMe File :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**